README.md

```diff
@@ -23,8 +23,9 @@ Documentation to the current _master_ branch as well as all releases can be foun
 
 If you encounter any issues while using Strimzi, you can get help using:
 
-- [Strimzi mailing list](https://www.redhat.com/mailman/listinfo/strimzi)
-- [Strimzi Slack workspace](https://join.slack.com/t/strimzi/shared_invite/enQtMzU2Mjk3NTgxMzE5LTYyMTUwMGNlMDQwMzBhOGI4YmY4MjhiMDgyNjA5OTk2MTFiYjc4M2Q3NGU1YTFjOWRiMzM2NGMwNDUwMjBlNDY)
+- [#strimzi channel on CNCF Slack](https://slack.cncf.io/)
```

Comment:
ditto as above.

Reply:
That links is useless unless you are already registered. So I think we should keep what I have there.